packages/genesys-spark-components/src/components/beta/gux-avatar/example.html

```diff
@@ -479,6 +480,41 @@ <h2>Notifications</h2>
   size="xsmall"
 >
 </gux-avatar-beta>
+<h2>UC Integration App Badge</h2>
+<gux-avatar-beta
+  name="Stephen Byrne"
+  presence="available"
+  label="Available"
+  badge
+  ring
+  notifications
+  uc-integration="teams"
+>
+</gux-avatar-beta>
+<gux-avatar-beta
+  name="Stephen Byrne"
+  presence="available"
+  label="Available"
+  badge
+  uc-integration="zoom"
+>
+</gux-avatar-beta>
+<gux-avatar-beta
+  name="Stephen Byrne"
+  presence="available"
+  label="Available"
+  ring
+  uc-integration="8x8"
+>
+</gux-avatar-beta>
+<gux-avatar-beta
+  name="Stephen Byrne"
+  presence="available"
+  label="Available"
+  ring
+  uc-integration="slack"
```

Comment:
Slack is not displaying anything.

Reply:
Sorry, I just put slack in as an example of an invalid app. I have changed it to be clearer. There is no Slack logo requirement.